st2common/st2common/models/db/__init__.py

```diff
@@ -114,8 +114,6 @@ def db_ensure_indexes():
         class_name = model_class.__name__
 
         # Note: We need to ensure / create new indexes before removing extra ones
-        LOG.debug('Ensuring indexes for model "%s"...' % (model_class.__name__))
```

Comment:
Yep, I removed it because it makes no sense to me to announce some action if we're intending to respond with the result of it the very next line.